exporter/datadogexporter/metrics_exporter_test.go

```diff
@@ -286,6 +287,67 @@ func Test_metricsExporter_PushMetricsData(t *testing.T) {
 			expectedSketchPayload: nil,
 			expectedErr:           nil,
 		},
+		{
+			metrics: createTestMetricsWithStats(),
+			source: source.Source{
+				Kind:       source.HostnameKind,
+				Identifier: "test-host",
+			},
+			histogramMode: HistogramModeDistributions,
+			hostTags:      []string{"key1:value1", "key2:value2"},
+			expectedSeries: map[string]interface{}{
```

Comment:
nit: feel free to use `any` here if you feel like it :)

Reply:
Nice! :) I like that. We should probably change that throughout though.